Add `osu!mania LN Tournament 4 Concludes` news post (#14072)
ppy/osu-wiki: news/2025/2025-12-08-osumania-ln-tournament-4-concludes.md

---
layout: post
title: osu!mania LN Tournament 4 Concludes
date: 2025-12-08 12:00:00 +0000
series: community_tournaments
---

> As o!mLN4 fades into cherished memories, we can reflect on everything that happened in the course of the tournament. With mind-shattering player performances to a plethora of new original music releases, the noodle galore ends off with a bang.

Reviewer: ```suggestion
As o!mLN4 fades into cherished memories, we can reflect on everything that happened in the course of the tournament. From mind-shattering player performances to a plethora of new original music releases, the noodle galore ends off with a bang.
```

The match began with both players opting with safer options within the pool, competing on what you could call the last real consistency battles on this pool. The match opened with ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267)'s selection, [Code Name: Sierra (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2455623#mania/5366692) (DE1). As one of the easier maps, the back and forth between both players trading misses with each other amounted to a small but significant 364 points to ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267). This was determined by literally a miss in the last 20 notes!

This set the first point to a 5-point streak, where ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267) took convincing leads by steering clear of misses and leveraging his consistency level, including a pick on the notoriously difficult chart of the pool, [Dead Aire](https://osu.ppy.sh/beatmapsets/2455636#mania/5366710) (TE4).

> However, ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805) showed up with a few tricks under his sleeve. Utilising his protects and performing well at a vital time, ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805) managed to dominate by 23,745 points on [See you soon](https://osu.ppy.sh/beatmapsets/2455632#mania/5366705) (CO1) and won another point by a 7,127 score difference on [TIMING HELL \~The Perpetual Chaotically Unsynced Pendulum\~](https://osu.ppy.sh/beatmapsets/2455661#mania/5366777) (CO3) with a lucky full combo.

Reviewer: wrong idiom wording
```suggestion
However, ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805) showed up with a few tricks up his sleeve. Utilising his protects and performing well at a vital time, ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805) managed to dominate by 23,745 points on [See you soon](https://osu.ppy.sh/beatmapsets/2455632#mania/5366705) (CO1) and won another point by a 7,127 score difference on [TIMING HELL \~The Perpetual Chaotically Unsynced Pendulum\~](https://osu.ppy.sh/beatmapsets/2455661#mania/5366777) (CO3) with a lucky full combo.
```

> Despite the 2 points, it seemed that ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805)'s physical cap had caught on. He eventually conceded 2 points to ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267) by score differences of 6,085 and 10,191 on [Lemming Ming (kemu Remix)](https://osu.ppy.sh/beatmapsets/2040875#mania/4257678) (TE2) and [Oxalis (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2455631#mania/5366703) (DE2) respectively, clinching 3rd place in the tournament and allowing ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267) to face ::{ flag=US }:: [Nepijin](https://osu.ppy.sh/users/11734610) one more time in the championship.

Reviewer: ```suggestion
Despite the 2 points, it seemed that ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805)'s physical cap had caught up to him. He eventually conceded 2 points to ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267) by score differences of 6,085 and 10,191 on [Lemming Ming (kemu Remix)](https://osu.ppy.sh/beatmapsets/2040875#mania/4257678) (TE2) and [Oxalis (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2455631#mania/5366703) (DE2) respectively, clinching 3rd place in the tournament and allowing ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267) to face ::{ flag=US }:: [Nepijin](https://osu.ppy.sh/users/11734610) one more time in the championship.
```

With ::{ flag=CL }:: [myucchii](https://osu.ppy.sh/users/10072733)'s usual habit of catching up in the 2nd half of the match, the turning of the tables felt imminent. However, ::{ flag=TH }:: [konkawe](https://osu.ppy.sh/users/15665805) managed to gather the match point on ::{ flag=CL }:: [myucchii](https://osu.ppy.sh/users/10072733)'s pick, [Last Journey (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2452231#mania/5357229) (WC2), scoring 991,088 with a full combo, securing their position in the stars as the top 3 of o!mLN4.

### Semifinals: ::{ flag=GB }:: [spanner dude](https://osu.ppy.sh/users/12489832) vs. ::{ flag=PH }:: [Plana_](https://osu.ppy.sh/users/24776314)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/ThhTCJ65hWQ" frameborder="0" allowfullscreen></iframe>
</div>

> This losers' bracket Semifinals match might be one of the most interesting in terms of dynamics, where both players brought to the table a difference of physicality versus consistency, and had a fight to the death for it.

Reviewer: reads better with the sentence right after it
```suggestion
This losers' bracket Semifinals match might be one of the most interesting in terms of dynamics, where both players brought to the table a clash between physicality versus consistency, and had a fight to the death for it.
```
